pom.xml

```diff
@@ -109,11 +109,12 @@
         <maven.templating.version>1.0.0</maven.templating.version>
         <maven.versions.version>2.7</maven.versions.version>
 
-        <sonar.jacoco.reportPaths>
-            ../target/jacoco-merge.exec,
-            ../../target/jacoco-merge.exec,
-            ../../../target/jacoco-merge.exec
-        </sonar.jacoco.reportPaths>
+        <sonar.coverage.jacoco.xmlReportPaths>
+            ../distribution-core/target/site/jacoco-aggregate/jacoco.xml,
+            ../../distribution-core/target/site/jacoco-aggregate/jacoco.xml,
+            ../../../distribution-core/target/site/jacoco-aggregate/jacoco.xml
```

Comment:
What is the most robust to a change: `../target/site/jacoco-aggregate/jacoco.xml` vs `../distribution-core/target/site/jacoco-aggregate/jacoco.xml`

Reply:
As discussed, the report is generated by jacoco:report-aggregate defined in the pom.xml of distribution-core. Therefore by default, the report is generated in distribution-core/target/...

We could change the outputDirectory of the jacoco:report-aggregate to output it in the target of the top project, but we prefer to use the default conf of jacoco:report-aggregate to have less surprise for developers looking for these reports